test/expr.jl

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Why is this in a new file? This can probably go to `parse` or `replutil` or `reflection`.